### ISSUE TITLE
The default lrbd.conf should have LUN, UUID and Authentication configuration

### ISSUE THREAD

**Issue by Martin-Weiss** [SUSE/DeepSea#532]
To have "security per default" - the default lrbd.conf should have authentication for discovery and tpg set i.e. like this:

`
    "auth": [
        {
            "target": "iqn.2016-11.org.linux-iscsi.igw.x86:sn.demo",
            "authentication": "tpg",
            "discovery": {
                "auth": "enable",
                "userid": "iSCSIDiscoveryUser",
                "password": "iSCSIDiscoveryPassword"
            },
            "tpg": {
                "userid": "iSCSIConnectUser",
                "password": "iSCSIConnectPassword"
            }
        }
    ],
`
Each image in the lrbd.conf should have a static and unique LUN and WWID configured:

`
                        {
                            "image": "demo",
                            "lun": "0",
                            "uuid": "090febcc-ee61-414b-bac0-0a53fd928762"
                        },
`

Could we get this added to the default lrbd.conf generator?

**Comment by swiftgist** [comment]
Do you mean the demo example in DeepSea?  Or the the lrbd samples?  Or what is generated by openATTIC?

I wouldn't think the samples since each file was trying to focus on one thing.

**Comment by Martin-Weiss** [comment]
> Do you mean the demo example in DeepSea?

Yes - the demo that is created in DeepSea is what I would like to have expanded with this issue.

For the same reasons we have to add LUN / UUID settings also per default in openATTIC - but I assume we have to create an issue for that against openATTIC..

**Comment by rjfd** [comment]
@Martin-Weiss 

> For the same reasons we have to add LUN / UUID settings also per default in openATTIC - but I assume we have to create an issue for that against openATTIC..

oA already generates LUN / UUID by default, and allows the user to edit those settings if the user wants.

**Comment by Martin-Weiss** [comment]
> oA already generates LUN / UUID by default, and allows the user to edit those settings if the user wants.

Great - then we just need to adjust the default lrbd.conf that comes via deepSea..
